manim_slides/slide.py

```diff
@@ -54,6 +54,14 @@ def __init__(
         self.__loop_start_animation: Optional[int] = None
         self.__pause_start_animation = 0
 
+    @property
+    def __background_color(self) -> Tuple[int, int]:
```

Comment:
```suggestion
    def __background_color(self) -> str:
```

The return type is not `Tuple[int, int]` :-)

Reply:
True, I forgot the copypaste. Shouldn't the return hint be pydantic.color.Color?

Reply:
Okay, nevermind, I just realized that you proposed to return .hex directly inside the property. Gotit!